Code/Framework/AzQtComponents/AzQtComponents/Components/Widgets/BreadCrumbs.cpp

```diff
@@ -401,6 +403,19 @@ namespace AzQtComponents
 
     bool BreadCrumbs::eventFilter(QObject* obj, QEvent* ev)
     {
+        if (obj == m_labelEditStack && isEditable())
+        {
+            switch (ev->type())
+            {
+            case QEvent::HoverEnter:
+                m_labelEditStack->setCursor(Qt::IBeamCursor);
+                return true;
+            case QEvent::HoverLeave:
+                m_labelEditStack->setCursor(Qt::ArrowCursor);
+                return true;
+            default:;
+            }
+        }
```

Comment:
This shouldn't be necessary, the `QLineEdit` gets the ibeamcursor by default already. Is there something else overriding the cursor?

Reply:
Well the `QLineEdit` does get the ibeamcursor by default, but it doesn't show up when the user mouses over the `QStackedWidget` because the current widget is a `QLabel` until the user clicks on the `QStackedWidget`. 

This change allows the `QStackedWidget` to display the ibeamcursor if the breadcrumbs are editable

Reply:
Setting the stacked widget's cursor to IBeam works really well, I've changed it to that! Thanks